libs/util.js

```diff
@@ -9,11 +9,12 @@ var Project = require('../models/project');
 exports.isLoggedIn = function (req, res, next) {
   // if user is authenticated in the session, carry on
   if (req.isAuthenticated()) {
-    return next();
+    next();
+  }
+  else {
```

Comment:
Not sure if such formatting is fine for eslint.
How about:
if {
**} else {**
}